### DISCUSSION TITLE
Migrate settings from Dev to Production Cluster

### DISCUSSION THREAD

**Discussion by richard-scott** [emqx/emqx#9200] [General]
I have been configuring a Dev Cluster and now want to export those setting so I can re-import them into the Production Cluster. Better still, I would also like to export the Production Settings and perform a diff on those to see what has changed.

Is there a way to do this?

I have tried looking via the Dashboard, but not all settings I've previously changed get loaded when I go back to the Dashboard to look and I end up with errors like this:
<img width="1475" alt="Screenshot 2022-10-21 at 10 01 55" src="https://user-images.githubusercontent.com/5149834/197157466-20de70d9-b154-407c-921d-c9c350affb4d.png">

I have to re-set ALL my previously set settings to resolve this error each and every time I visit this section.

**Comment by lafirest** [comment]
@richard-scott Hi, Do you see if this [Config and data backup & restore](https://www.emqx.io/docs/en/v4.3/advanced/config-and-data-backup-restore.html#terminologies) helps you?
